content/_data/changelogs/weekly.yml

```diff
@@ -2352,6 +2352,35 @@
     # pull: 3309 too minor (form target fix for admin monitor)
     # pull: 3334 too minor (German translation)
     # pull: 3352 too minor (ExtensionList#lookupSingleton)
+- version: "2.113"
+  date: 2018-03-25
+  changes:
+    - type: major rfe
+      pull: 3359
+      message: >
+        Update Remoting from 3.18 to 3.19 so that Jenkins core can always deserialize exceptions even if they're not whitelisted.
+        To benefit from this improvement, Remoting needs to be updated on the agent side as well.
+      references:
+        - issue: 50237
+        - issue: 49618
+        - url: https://github.com/jenkinsci/remoting/blob/master/CHANGELOG.md#319
+          title: full changelog
+    - type: bug
+      pull: 3359
+      issue: 50237
+      message: >
+        JEP-200: Whitelist <code>org.apache.tools.ant.Location</code> to prevent deserialization exception when listing agent files in non-existent directory or invalid filter.
+    - type: bug
+      issue: 46386
+      pull: 3362
+      message: >
+        Prevent some cases of linkage errors relating to Servlet classes when code is run on an agent.
```

Comment:
"... when System Properties are retrieved on an agent"?

Reply:
Irrelevant for the target audience, I think. (Even this one is borderline useless, but at least "linkage error" would show up in a log somewhere.)